### ISSUE TITLE
Confusing empty output when ran against hidden file

### ISSUE THREAD

**Issue by HipyCas** [The-Noah/lsfp#31]
**Describe the bug**
When you run lsfp against a hidden file, for example when running `lsfp .editorconfig`, no output is provided, which leads to confusion as you do not know if the file does not exist or what is going on at all. The problem is caused by the following code in the `print_item` function:

```rust
if !flags.all && file_detection::is_hidden(&path) {
    return;
}
```

As far as I understand, the problem comes when this function is called on a single hidden file, as it returns void, causing the main function to print no content to the screen. A solution may be passing to this function an additional parameter, which evaluate to true if the code is being ran against a single file, then the function returning as if the file was not hidden and therefore getting a proper output.

**To Reproduce**
Run the code against any single file which is hidden. This does not happen with a file that is not hidden, or with a hidden directory (prints content):

```bash
$ lsfp .gitignore

$ lsfp Cargo.toml
Cargo.toml

$ lsfp .vscode
extensions.json
settings.json
```

**Expected behavior**
Probably the best would be to just print the file as if was not hidden, but I am sure there are many other possibilities.

**Additional context**
I have never written any Rust code in my life, except when I added some licenses, but that was not really coding. I would be happy to take this as my first Rust code and challenge if you are fine with that! 😄

**Comment by The-Noah** [comment]
Sure, go right ahead! I appreciate your help.

**Comment by HipyCas** [comment]
This issue was solved in pull request #33
